core/examples/hello_world.rs

```diff
@@ -2,21 +2,12 @@
 //!  This example shows you how to define ops in Rust and then call them from
 //!  JavaScript.
 
-use deno_core::op;
-use deno_core::Extension;
-use deno_core::JsRuntime;
-use deno_core::Op;
-use deno_core::RuntimeOptions;
-
-// This is a hack to make the `#[op]` macro work with
-// deno_core examples.
-// You can remove this:
 use deno_core::*;
```

Comment:
This is a more convenient import for most of the examples, so leaving it here (it's no longer required though)